config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do  
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+ get 'users/:user_id/posts', to: 'posts#index', as: 'user_posts'
+ get 'users/:user_id/posts/:id', to: 'posts#show', as: 'user_post'
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+ get 'users', to: 'users#index', as: 'users'
+ get 'users/:id', to: 'users#show', as: 'user'
+
+ root 'users#index'
```

Comment:
- [ ] Its a requirement that we dont expose any unecessary endpoints. In order to acieve this we can use the `:only` option to specify only the routes you want to include:
e.g
```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Kindly fix this